src/pods/generate/SubdomainIdentifierGenerator.ts

```diff
@@ -0,0 +1,21 @@
+import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
+import { ensureTrailingSlash, extractScheme } from '../../util/PathUtil';
+import type { IdentifierGenerator } from './IdentifierGenerator';
+
+/**
+ * Generates identifiers by using the slug as a subdomain on the base URL.
```

Comment:
I wonder if the term "slug" would cause confusion with the `Slug` header and its usage. "Pod identifier" or so, is that what we mean?